client/flutter/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,36 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  desc "Push a new beta build to TestFlight"
+  lane :beta do
+    build_app(workspace: "Runner.xcworkspace", scheme: "Runner")
+    upload_to_testflight
+  end
+
+  desc "Promote the beta build to App Store"
+  lane :submit_review do
+    deliver(
+      submit_for_review: true,
```

Comment:
Realistically on the first release, setting `submit_for_review` to `true` will likely fail, as AppStore Connect has follow-up questions around encryption and ad tracking. I'd recommend doing at least one release manually before automating the actually `submit_for_review`

Reply:
Totally agree.